demisto_sdk/commands/lint/tests/test_pylint_plugin/certified_partner_level_checker_test.py

```diff
@@ -0,0 +1,328 @@
+import astroid
+import pylint.testutils
+from demisto_sdk.commands.lint.resources.pylint_plugins import \
+    certified_partner_level_checker
+
+# You can find documentation about adding new test checker here:
+# http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#write-a-checker
+
+
+class TestSysExitChecker(pylint.testutils.CheckerTestCase):
+    """
+    Class which tests the functionality of sys exit checker .
+    """
+    CHECKER_CLASS = certified_partner_level_checker.CertifiedPartnerChecker
+
+    def test_sys_exit_exists(self):
+        """
+        Given:
+            - String of a code part which is being examined by pylint plugin.
+        When:
+            - sys.exit exists in the code.
+        Then:
+            - Ensure that the correct message id is being added to the message errors of pylint
+        """
+        _, node_b, _ = astroid.extract_node("""
+            def test_function(): #@
+                sys.exit(1) #@
+                return True #@
```

Comment:
what is the usage of `#@`?

Reply:
#@ is used from pylint testutils(which those tests inherent from) and means that this row will be evaluated( which means we will have the type of node of this row as a return value from astroid.extract_node